nodogsplash2/files/nodogsplash.init

```diff
@@ -248,10 +248,42 @@ create_instance() {
   procd_close_instance
 }
 
+depends() {
+  if [ "$1" = "iptables" ] ; then
+    if [ $(uci get nodogsplash.@nodogsplash[0].fwhook_enabled) = "1" ] ; then
+      if $WD_DIR/ndsctl status > /dev/null; then
+       echo " * Restarting NodogSplash" 
+       /etc/init.d/nodogsplash restart
+      fi
+    else   echo " * NodogSplash fwhook is disabled"
+    
+    fi
```

Comment:
Instead of this:
```
  else   echo " * NodogSplash fwhook is disabled"

  fi
```
do
```
  else
    echo " * NodogSplash fwhook is disabled"
  fi
```

Reply:
done